app/controllers/reviews_controller.rb

```diff
@@ -9,8 +9,11 @@ def new
 
   def create
     @quantity_collection = [[1, 1], [2, 2], [3, 3], [4, 4], [5, 5]]
+    updated_params = review_params
+    updated_params[:user_id] = current_user.id
+
     @venue = Venue.find(params[:venue_id])
-    @review = @venue.reviews.new(review_params)
+    @review = @venue.reviews.new(updated_params)
```

Comment:
Delete lines 12-14, revert line 16 to `@review = @venue.reviews.new(review_params)`. The next line `@review.user = current_user` will set the user id to the review.